JASP-Engine/JASP/R/auditClassicalBenfordsLaw.R

```diff
@@ -115,16 +115,16 @@ auditClassicalBenfordsLaw <- function(jaspResults, dataset, options, ...){
     
     } else if (options[["distribution"]] == "uniform"){
 
       procedureText <- base::switch(options[["digits"]],
-                                    "first" = gettextf("The uniform distribution assigns equal probability to all values that may occur. The goal of this procedure is to determine to which extent the first leading numbers in the population follow the uniform distribution, and to test this relation with <b>%1$s</b> confidence. Supposedly random data that do not conform to the uniform distribution might need further verification.", confidenceLabel),
-                                    "firstSecond" = gettextf("The uniform distribution assigns equal probability to all values that may occur. The goal of this procedure is to determine to which extent the first two leading numbers in the population follow the uniform distribution, and to test this relation with <b>%1$s</b> confidence. Supposedly random data that do not conform to the uniform distribution might need further verification.", confidenceLabel),
-                                    "last" = gettextf("The uniform distribution assigns equal probability to all values that may occur. The goal of this procedure is to determine to which extent the last numbers in the population follow the uniform distribution, and to test this relation with <b>%1$s</b> confidence. Supposedly random data that do not conform to the uniform distribution might need further verification.", confidenceLabel))
+                                    "first" = gettextf("The uniform distribution assigns equal probability to all values that may occur. The goal of this procedure is to determine to what extent the first leading digits in the data set follow the uniform distribution, and to test this relation with <b>%1$s</b> confidence. Supposedly random data that do are not uniformly distributed might need further investigation.", confidenceLabel),
+                                    "firstSecond" = gettextf("The uniform distribution assigns equal probability to all values that may occur. The goal of this procedure is to determine to what extent the first two leading digits in the data set follow the uniform distribution, and to test this relation with <b>%1$s</b> confidence. Supposedly random data that do are not uniformly distributed might need further investigation.", confidenceLabel),
+                                    "last" = gettextf("The uniform distribution assigns equal probability to all values that may occur. The goal of this procedure is to determine to what extent the last digits in the data set follow the uniform distribution, and to test this relation with <b>%1$s</b> confidence. Supposedly random data that do are not uniformly distributed might need further investigation.", confidenceLabel))
```

Comment:
"The uniform distribution assigns equal probability to all possible last digits that may occur. The goal of this procedure is to determine to what extent the first leading digits in the data set follow the uniform distribution, and to test this relation with a type I error of <b>%1$s</b>. If the uniform distribution assumption is reasonable/desirable, then violation of uniformity is cause for further investigation."